src/components/Input/StringInput.tsx

```diff
@@ -14,8 +14,11 @@ export interface Props extends TextInputProps {
  * the value on ESCAPE.
  */
 export function StringInput({ onEnter, value, errorCheck, ...props }: Props) {
-  const { value: storedValue, set: setStoredValue } =
-    usePropListeningState<string>(value);
```

Comment:
Guessing that the changes in the Input component swill disappear if you merge in master in this branch?